lib/ical/parse.js

```diff
@@ -232,9 +232,20 @@ ICAL.parse = (function() {
     var multiValue = false;
     var structuredValue = false;
     var propertyDetails;
+    var splitName;
+    var ungroupedName;
+
+    // fetch the ungrouped part of the name
+    if (name.indexOf('.') !== -1 && state.designSet.propertyGroups) {
```

Comment:
Minor nit, let's turn this around. Slight perf increase in skipping the indexOf for iCalendar:
```suggestion
    if (state.designSet.propertyGroups && name.indexOf('.') !== -1) {
```